### PR TITLE
fix(electron): close app while on fullscreen should be minimize instead

### PR DESCRIPTION
**Pull request by pengx17**

### CODE REVIEW

**Comment by pengx17** [comment]
* **#6534** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6534?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/electron/src/main/main-window.ts [hunk 1]
```diff
@@ -127,7 +127,11 @@ async function createWindow(additionalArguments: string[]) {
       //   - all browser windows will capture the "close" event
       //   - the hidden window will close all windows
       //   - "window-all-closed" event will be emitted and eventually quit the app
-      browserWindow.hide();
+      if (browserWindow.isFullScreen()) {
+        browserWindow.setFullScreen(false);
```
Comment (EYHN) [inline packages/frontend/electron/src/main/main-window.ts:134]: can we do both `setFullscreen(false)`  and `hide()`

Comment (pengx17) [reply packages/frontend/electron/src/main/main-window.ts:134]: It seems setting full screen to false will trigger an animation. During animation transitioning period, `hide` will not work.

Comment (EYHN) [reply packages/frontend/electron/src/main/main-window.ts:134]: https://github.com/electron/electron/issues/6033#issuecomment-242023295

seems to be a solution

[confused](https://app.graphite.dev/settings/meme-library?org=toeverything)

**Review by EYHN — APPROVED** [review]

**Pull request merged**